oracle/Assets/Scripts/StatisticsBox.cs

```diff
@@ -68,6 +68,44 @@ public void removeEntry(string name)
         }
     }
 
+    /// <summary>
+    /// Adds a toggle into the UI. If one already exists,
+    /// then the value will be updated.
+    /// </summary>
+    public void addToggle(string name, bool value)
+    {
+        var toggleTransform = gameObject.transform.Find(name);
+        Toggle toggle;
+        if (toggleTransform != null)
+        {
+            toggle = toggleTransform.gameObject.GetComponent<Toggle>();
+        }
+        else
+        {
+            var toggleObject = new GameObject(name);
+            toggleObject.transform.parent = gameObject.transform;
+
+            toggle = toggleObject.AddComponent<Toggle>();
+            //var toggleText = toggle.GetComponentInChildren<Text>();
+            //toggleText.font = font;
+            //toggleText.fontSize = fontSize;
+        }
+
+        toggle.isOn = value;
+    }
```

Comment:
ran into issues setting up a ui toggle component w/in the stat box. found some better details [here](https://stackoverflow.com/questions/41194515/unity-create-ui-control-from-script)

Reply:
room status is currently displayed using the existing logic for adding text entries